src/main/resources/templates/db_scripts/schema.json

```diff
@@ -1,10 +1,28 @@
 {
+  "scripts": [
+    {
+      "run": "after",
+      "snippet": "CREATE OR REPLACE VIEW invoice_lines_view AS SELECT invoice_lines.id AS id, invoice_lines.jsonb AS jsonb, jsonb_build_object('acqUnitIds', invoices.jsonb->'acqUnitIds') || invoice_lines.jsonb AS metadata FROM invoice_lines LEFT JOIN invoices ON (invoice_lines.jsonb -> 'invoiceId'::text) = (invoices.jsonb -> 'id'::text);",
```

Comment:
The default RMB's approach to build views (i.e. by using `"views"` property instead of script snippet) could be used but in this case the search will be based on entire invoice content but not restricted to `acqUnitIds` property. So script snippet is fine here if there are no plans to search by invoice content later...

Reply:
At this stage of development I'm fine with either way - it's not a ton of work to change it later on.  That said, I'm not sure I really see the benefit of building a custom json object unless you're going to be returning it in queries... Is it more efficient to use jsonb_build_object than a `||` merge?  

I say that with the assumption that most queries will ask for the jsonb column, and query against the metadata column.  Technically there's nothing stopping you from querying and asking for the metadata column, but I don't think we're doing that anywhere in acquisitions.

Reply:
> Is it more efficient to use jsonb_build_object than a `||` merge?
The `jsonb_build_object` is used to create jsonb from a set of invoice parameters (for now it's only `acqUnitsIds`) that we will use for searching, then concatenate it to invoice_line.jsonb using `||`, and honestly, I don't know any other easy way to merge two jsonb.

Reply:
My point was you could just merge the entire invoice.jsonb and invoice-line.jsonb right?  IIRC this is what we do in orders.  I'm not sure picking just the acqUnits field from invoice.json is really necessary.

Reply:
This is a reason why we raised a question if there is okay to allow to search by all available fields or only by limited set. As Ann-Marie clarified, only limited set is going to be provided. So @pautau-s implemented as clarified. If it is okay to let logic to allow filter by all properties, then probably we do not need these views at all :) [RMB v26.3.0](https://github.com/folio-org/raml-module-builder/releases/tag/v26.3.0) allows to use foreign keys to build desired queries ([RMB-395](https://issues.folio.org/browse/RMB-395))...